flytekit/annotated/base_task.py

```diff
@@ -230,7 +241,7 @@ def __init__(
 
     # TODO lets call this interface and the other as flyte_interface?
     @property
-    def python_interface(self):
+    def python_interface(self) -> Interface:
```

Comment:
This is supposedly the Override of the parent's python_interface? should the return be consistent?

Reply:
It should but I'd rather it be accurate than consistent with the base class.